HW_2/task_03.py

```diff
@@ -0,0 +1,23 @@
+#Напишите программу, которая принимает две строки вида “a/b” - дробь с числителем и знаменателем. 
+#Программа должна возвращать сумму и произведение* дробей. 
+#Для проверки своего кода используйте модуль fractions.
+import fractions
+x = '1/3'
+y = '1/2'
```

Comment:
Поменять на 
```suggestion
x = '1/2'
y = '1/2'
```
Ответы не сходятся
![image](https://github.com/annkrv80/for_HW_Python/assets/49741194/34dcf831-baef-40a1-b95c-c61df75f6290)